docs/deploy-development-rp.md

```diff
@@ -76,42 +76,108 @@
+  ```bash
+  INFO[2022-04-01T10:02:41-05:00]pkg/util/cluster/cluster.go:318 cluster.(*Cluster).Create() creating cluster complete
+  ```
 
-   OR use the create utility:
+## Steps to connect to the Cluster and confirm it is up via kubectl or oc:
 
+1. At your terminal execute to create the admin.kubeconfig locally. This will allow you to connect to the cluster via kubectl or oc
    ```bash
-   CLUSTER=cluster go run ./hack/cluster create
+   CLUSTER=<aro-cluster-name> make admin.kubeconfig
    ```
 
-   Later the cluster can be deleted as follows:
+1. Disconnect from "rp-vnet" vpn and connect to "dev-vnet" vpn. The steps are identical to connecting to "rp-vnet" in the #preparation-to-create-cluster section. You will just need to download the dev-vpn client certificate locally, create the VPN connection using your VPN client of choice, and use nmap to get the IP of the internal load balancer from the <aro-cluster-name-rg>. You can find this address at: https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/<subscriptionid>/resourceGroups/<aro-cluster-rp-rg>/providers/Microsoft.Network/loadBalancers/<aro-cluster-name>-<random string for your lb>-internal/frontendIpPool (internal-lb-ip-v4). In my case the IP was 10.62.174.
+```bash
+nmap -p 6443 -sT 10.62.174.4 -Pn
+Starting Nmap 7.92 ( https://nmap.org ) at 2022-04-01 10:36 CDT
+Nmap scan report for 10.62.174.4
+Host is up (0.070s latency).
 
-   ```bash
-   CLUSTER=cluster go run ./hack/cluster delete
-   ```
+PORT STATE SERVICE
+6443/tcp open sun-sr-https
 
-   [1]: https://docs.microsoft.com/en-us/azure/openshift/tutorial-create-cluster
+Nmap done: 1 IP address (1 host up) scanned in 0.14 seconds
+```
+1. Update admin.kubeconfig cluster.server parameter to use this IP as well. It should look like this:
+```bash
+server: https://<ip>:6443
+```
+1. Updated your kubeconfig env var to point to the admin.kubeconfig
+```bash
+export KUBECONFIG=$(pwd)/admin.kubeconfig
+```
+1. Execute a kubectl (or oc) command to see if you can list any K8s objects
+```bash
+kubectl get nodes --insecure-skip-tls-verify
+```
+2. You should see something like this. If so, your cluster is up!
+```bash
+NAME                                        STATUS   ROLES    AGE    VERSION
+cdp-cfs-eleven-bljdk-master-0               Ready    master   3h7m   v1.22.3+4dd1b5a
+cdp-cfs-eleven-bljdk-master-1               Ready    master   3h6m   v1.22.3+4dd1b5a
+cdp-cfs-eleven-bljdk-master-2               Ready    master   3h6m   v1.22.3+4dd1b5a
+cdp-cfs-eleven-bljdk-worker-eastus1-2r9b4   Ready    worker   177m   v1.22.3+4dd1b5a
+cdp-cfs-eleven-bljdk-worker-eastus2-jgrj9   Ready    worker   177m   v1.22.3+4dd1b5a
+cdp-cfs-eleven-bljdk-worker-eastus3-fd646   Ready    worker   177m   v1.22.3+4dd1b5a
+```
 
-1. The following additional RP endpoints are available but not exposed via `az
-   aro`:
+## RP endpoints are available but not exposed via `az aro`:
```

Comment:
Removing "the following" made the purpose of this heading confusing. I had to look at the diff to figure out what it's saying.

Reply:
```suggestion
## Available RP endpoints not exposed via `az aro`
```